### ISSUE TITLE
About network

### ISSUE THREAD

**Issue by Z-Z-J** [Shimingyi/MotioNet#12]
` for stage_index in range(0, stage_number):
            for conv_index in range(len(kernel_size_set)):
                layers.append(
                    nn.Sequential(
                        nn.Conv1d(channel, channel, kernel_size_set[conv_index], stride_set[conv_index], dilation=1, bias=True),
                        nn.BatchNorm1d(channel, momentum=0.1)
                    )
                )
        self.stage_layers = nn.ModuleList(layers)`

stage_number : 2  len(kernel_size) : 3         2*3 = 6
![2](https://user-images.githubusercontent.com/53818279/93056186-a9aa5700-f69e-11ea-80d7-ad03c9ddfe6f.png)

At the end of your paper,  ( Conv + BatchNorm + LReLU + Dropout ) or (Conv + BatchNorm + LReLU + Dropout + Adap AP) only have three layers ,not six layers. 
Why?

**Comment by Z-Z-J** [comment]
Another question !!    In  pooling_shrink_net(base_model):
self.expand_conv = nn.Conv1d(in_features, channel, kernel_size=3, stride=2, bias=True)

However, in your paper, kernel_size = 1 , stride =1 , channel = 2048(not 1024,in your code) !!

I look forward to your answer!!

**Comment by Shimingyi** [comment]
Thanks for your question, in here we use a little different settings with paper's. 
- In paper we just show the block strcuture, in one block there are three layers, but in experiments you can set different block numbers. And also, 1 stage(3 layer) is also works here, you can also use 1 stage to get same performance like h36m data. But 2 stage setting is rubuster in more cases so I set 2 as default number.
- When you increase the channel number to 2048, the error will be lower around 52mm like what we show in the paper, but it requires more training time and model file storage(around 600mb rather than 100mb currently). So in the opensource version, I want to find a balance between performance and user-convenience so I choose the number 1024.
